recipe/meta.yaml

```diff
@@ -101,6 +135,19 @@ outputs:
         # host C compilers work fine (better) on mac
         - {{ compiler('c') }}
     test:
+      # Ensure the correct dependencies are available for testing
+      requires:      
+        - mpi 1.0 openmpi
+        - libevent
+        - libhwloc
+        - libnl  # [linux]
+        - libnuma  # [linux]
+        - libudev1  # [linux]
+        - numactl  # [linux]
+        - rdma-core  # [linux]
```

Comment:
Each dependency added to the test env is indicating that it is a dependency that _users_ must specify explicitly in order for this to work. I suspect this should really be _only_ ucc, but maybe I'm wrong?

Reply:
OK